pos_autoreconcile/models/point_of_sale.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 UAB Versada (<http://www.versada.lt>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    def _create_account_move_line(self, session=None, move=None):
+        to_ret = super(PosOrder, self)._create_account_move_line(
+            session=session, move=move)
+
+        account_def = self.env['ir.property'].get(
+            'property_account_receivable', 'res.partner')
+
+        grouped_data = {}
+
+        for order in session.order_ids:
+            current_company = order.sale_journal.company_id
+            order_account = (
+                order.partner_id and
+                order.partner_id.property_account_receivable_id.id and
+                order.partner_id.property_account_receivable_id.id or
+                account_def and account_def.id or
+                current_company.account_receivable.id
```

Comment:
```res.company``` model has no field named ```account_receivable```. And I think that the sale journal is not mandatory on a ```pos.order```.